modules/dr-recover-expired-control-plane-certs.adoc

```diff
@@ -5,167 +5,28 @@
 [id="dr-scenario-3-recovering-expired-certs_{context}"]
 = Recovering from expired control plane certificates
 
-Follow this procedure to recover from a situation where your control plane certificates have expired.
+OpenShift can now automatically recover control plane certificates, replacing the previous manual steps.
```

Comment:
* OpenShift -> {product-title}
* Exactly which z-stream(s) did this land for? We'll want to say something like:

```
As of 4.y.z, {product-title} can automatically recover control plane certificates. You no longer need to perform manual steps to recover from expired control plane certificates.
```

Though the second sentence might need tweaked, since it seems there are still some manual steps, in approving the CSRs - see my next comment.

And we might need to adjust this some for 4.5.

Reply:
A question here though, it says that it replaces the previous manual steps, but there is still some manual intervention needed, in approving the pending CSRs, correct?

Are we actually still recovering from expired control plane certs here, or are we rotating the certs before they expire?

Reply:
> Are we actually still recovering from expired control plane certs here, or are we rotating the certs before they expire?

No, this is after they are already expired. Kubelet rotates certs on its own fine, but if they expire it needs CSR approvals to recover.

I've updated the wording to make it clearer.